packages/medusa-core-utils/src/create-require-from-path.ts

```diff
@@ -12,7 +12,6 @@ const fallback = (filename: string) => {
 }
 
 // Polyfill Node's `Module.createRequireFromPath` if not present (added in Node v10.12.0)
-const createRequireFromPath =
-  Module.createRequire || Module.createRequireFromPath || fallback
+const createRequireFromPath = Module.createRequire || fallback
```

Comment:
Note: Checked again and [our documentation](https://github.com/medusajs/medusa/blob/master/docs/content/usage/create-medusa-app.mdx?plain=1#L32-L36) says that Medusa supports Node 14 and 16. This change should not introduce unexpected regressions.

Reply:
If I am not wrong, 17 and 18 works as well and since 18.14 is the LTS we should aim to support it especially that a lot of users are on those versions. Is it still supported ?

cc: @olivermrbl